### DISCUSSION TITLE
Override existing link automatically without showing the confirmation dialog

### DISCUSSION THREAD

**Discussion by hochbit** [keycloak/keycloak#34263] [Q&A]
Hi,

My Setup: 

Two Keycloak instances **without persistence,** one of them is the **id_provider  of the other**. (users are provided from external provider to the id_provider instance or can be assumed automatically created on startup/restart). 

When the id_provider instance gets reset (that means the database or realm is recreated), the client instance users couldn't not login to the id_provider instance.

I can resolve it with adding "Confirm override existing link" to my authorization (first broker login):

![authflow](https://github.com/user-attachments/assets/a5f439be-38f8-481d-972d-0d48457f3559)

So my question is, is somehow possible to **disable showing the confirmation dialog** to the user and override the link automatically?

Or are there any other ways possible, for achieving the desired outcome?

Thanks in advance.

**Comment by bogdannedelcu** [comment]
did you find any solution to this?

**Reply by hochbit** [reply]
Actually yes, we have our own theme. Put a little javascript into it. If it sees the button for it. It does click in immediately after page finishes to load ... user sees it just a split second. 

```javascript

window.onload = function() {
    const confirmButton = document.getElementById('confirmOverride');
    if (confirmButton) {
        confirmButton.click();
    }
};
```
